mysql-test/suite/innodb/t/purge_secondary.test

```diff
@@ -4,10 +4,6 @@
 SET @save_stats_persistent = @@GLOBAL.innodb_stats_persistent;
 SET GLOBAL innodb_stats_persistent = 0;
 
---disable_query_log
-call mtr.add_suppression("InnoDB: Difficult to find free blocks in the buffer pool");
```

Comment:
Yes, something tied to a test-and-set of `buf_pool.LRU_warned` should be a good idea. That warning would have to be suppressed globally in `mtr`, because some large tests would occasionally trigger it; see 967a148966a13fe7eb58232403a3c289abff91a8 for the message in `buf_flush_page_cleaner()` itself.

Reply:
As of d45b7f665513e5a8e24799157b64caa7c5685a03, `buf_pool.LRU_warn()` will be invoked either by the page cleaner thread or in `buf_LRU_get_free_block()`. The `buf_pool.LRU_warned` flag will be cleared by `buf_flush_page_cleaner()` when the buffer pool pressure is relieved.